sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -44,9 +44,10 @@
       :notDraft="notDraft"
     />
   </student-page-container>
-  <ConfirmEditApplication
+  <confirm-change-application
```

Comment:
As stated in the comment https://github.com/bcgov/SIMS/pull/3750#discussion_r1779233019 

We can leverage the ConfirmModel.vue to build this as per technical AC. 

To guide further, I constructed the new modal with ConfirmModal.vue which eliminates the need for building a modal from scratch. 

![image](https://github.com/user-attachments/assets/e0115dd6-ab58-476f-bdfe-0d9dccea53b0)


```vue
<confirm-modal
    title="Edit application"
    ref="editApplicationModal"
    okLabel="Submit"
    cancelLabel="No"
    :disable-primary-button="!conditionsAccepted"
    ><template #content>
      <div class="m-4">
        <v-checkbox
          color="primary"
          v-model="conditionsAccepted"
          label="I acknowledge that by editing my application, I may be required to
        resubmit previously approved exception requests. My institution may be
        required to resubmit program information, and my parent or partner may be
        required to resubmit supporting information."
          hide-details="auto"
        ></v-checkbox>
      </div>
      <div class="m-4">
        <banner
          v-if="isStudentEndDateWithinDeadline"
          :type="BannerTypes.Warning"
        >
          <template #content
            >Please note your application has now passed the six week deadline
            for completed applications to be received by StudentAid BC. All
            edits to your application will require additional review from
            StudentAid BC to be considered for funding. Please see the following
            link for information on the
            <a
              rel="noopener"
              target="_blank"
              href="https://studentaidbc.ca/sites/all/files/form-library/appeal_fundingafterenddate.pdf"
              >funding after end date appeal</a
            >.
          </template>
        </banner>
      </div>
    </template></confirm-modal
  >
```

Reply:
It looks like this with added spacing, please take a look and  make any changes required and please throw a quick snap to business. 
![image](https://github.com/user-attachments/assets/6e53c698-8ea9-4519-a040-4d9c6309a796)